status-im/status-react
# Added build scripts

### Summary:

As a developer I want an easy way to generate a production build on android.

### Steps to test:
- run `./scripts/build-android.sh prod`
- verify you get a working `apk`

status: ready
